### PR TITLE
Adds StatusCallBack feature for tracking sms

### PR DESCRIPTION
**Pull request by ctorresr**
Adds StatusCallBack feature for tracking sms by adding the method messageWithTracking in Twilio Class.

### CODE REVIEW

File: src/Dummy.php [hunk 1]
```diff
@@ -17,7 +18,7 @@ public function message($to, $message, $from = null)
     /**
      * @param string $to
      * @param string $message
-     * @param array $mediaUrls
+     * @param array  $mediaUrls
```
Comment (hannesvdvreken) [inline src/Dummy.php]: I won't merge these alignments. Please revert them.

**Comment by hannesvdvreken** [comment]
Sorry not going to merge this.

You can do 
```php
$twilio->getTwilio()->account->messages->sendMessage($from ?: $this->from, $to, $message, $mediaUrls, $trackingParams);
```
if you need to.

I'm planning on deprecating the optional `$from = null` field cause obviously it's in the way.
Next thing I'll do is use `func_get_args()` to get all function arguments, prepend the `$this->from` and invoke the `message` method. This way you can pass whatever you like and then I can also remove the `messageWithMedia` method cause obviously that inspired you to send this PR.

**Comment by hannesvdvreken** [comment]
If you want you can test out v3.0.0-RC1 which supports additional parameters to `->message` which will be passed on to the Twilio SDK.

Let me know what you think.

**Comment by ctorresr** [comment]
Thanks, as soon as I have time will test it out and give you feedback.

**Pull request closed without merging**